reference/release-notes/1.21.2.md

```diff
@@ -0,0 +1,29 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.21.2.
+
+Please see [component versions](https://anbox-cloud.io/docs/reference/component-versions) for a list of updated components.
+
+## New features & improvements
+
+### Core stack improvements
+
+* WiFi support for AAOS images.<!--AC-2211-->
+
+### Other
+
+* Android security updates for March 2024 (see [Android Security Bulletin - April 2024](https://source.android.com/docs/security/bulletin/2024-04-01) for more information).<!--AC-2391-->
```

Comment:
Android security updates for March 2024 
->
Android security updates for April 2024